alpha/apps/kaltura/lib/storage/kFileSyncUtils.class.php

```diff
@@ -2229,14 +2229,34 @@ public static function doAllEntryFlavorsExistInStorage($storageId, $flavorAssets
 	 */
 	public static function getReadyInternalFileSyncsForKey(FileSyncKey $syncKey)
 	{
-		$peridoicStorageFileSyncs = self::getFileSyncsFromPeriodicStorage($syncKey->getPartnerId(), $syncKey);
+		$periodicStorageFileSyncs = self::getFileSyncsFromPeriodicStorage($syncKey->getPartnerId(), $syncKey);
 
 		$c = FileSyncPeer::getCriteriaForFileSyncKey( $syncKey );
 		$c->addAnd ( FileSyncPeer::FILE_TYPE , FileSync::FILE_SYNC_FILE_TYPE_URL, Criteria::NOT_EQUAL);
 		$c->addAnd ( FileSyncPeer::STATUS , FileSync::FILE_SYNC_STATUS_READY );
 		$localfileSyncs = FileSyncPeer::doSelect( $c );
 
-		return array_merge($peridoicStorageFileSyncs, $localfileSyncs);
+		return array_merge($periodicStorageFileSyncs, $localfileSyncs);
+	}
+	
+	public static function getReadyRemoteFileSyncsForAsset($id, $asset, $objectType, $objectSubType)
+	{
+		$c = new Criteria();
```

Comment:
@yossipapi this method is called getReadyRemoteFileSyncsForAsset, so perhaps a DC check should be on the base $c query as well?